edgelessdb/getting-started/marblerun.md

```diff
@@ -0,0 +1,44 @@
+# Running EdgelessDB under MarbleRun
+
+If you want to run EdgelessDB as a distributed application in a confidential cluster, you can combine it with [MarbleRun](https://marblerun.sh).
+
+## Changes
+When you are running EdgelessDB as a Marble, it will work slightly different. Rather than generating an encryption key based on the SGX sealing key, MarbleRun will have to provide a key and a root certificate which EdgelessDB can use to sign its own ones. Ideally, you can use MarbleRun's [Secrets management](https://docs.edgeless.systems/marblerun/#/content/features/secrets-management) to accomplish this. Given now MarbleRun will handle EdgelessDB's certificate and key management, Recovery will be unavailable, as it is expected that this is handled on MarbleRun's level for the whole cluster and not just for a single EdgelessDB instance.
+
+## Setup
+### Launch the MarbleRun Coordinator
+Setup and launch the MarbleRun Coordinator by following the [quickstart guide](https://docs.edgeless.systems/marblerun/#/content/getting-started/quickstart) until Step 3. Alternatively, you can run the Coordinator without Kubernetes by following the [standalone deployment guide](https://docs.edgeless.systems/marblerun/#/content/deployment/standalone) until it is waiting for a manifest to be uploaded.
```

Comment:
Why not reference the https://docs.edgeless.systems/marblerun/#/contentdeployment/kubernetes/ instead of the quickstart?

Reply:
Changed it to https://docs.edgeless.systems/marblerun/#/content/deployment/cloud, it serves as a landing page for both cases, even though it is called "Cloud" deployment. I think we should add a general deployment page here, though basically the cloud one mainly serves this purpose.